### ISSUE TITLE
nn.MSELoss which computes the mean-squared error between the input and the target ??

### ISSUE THREAD

**Issue by birdmw** [pytorch/pytorch#17547]
https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html

You say "There are several different loss functions under the nn package . A simple loss is: nn.MSELoss which computes the mean-squared error between the input and the target."

don't you mean "between the OUTPUT and the target"

**Comment by soumith** [comment]
it's the input to the loss. it's the output of the network.
